theano/compile/tests/test_function_module.py

```diff
@@ -40,7 +40,8 @@ def test_none(self):
         fn = function([], None)  # ok
         rval = fn()
         if rval == []:
-            raise KnownFailureTest('See #254: Using None as function output leads to [] return value')
+            raise SkipTest("Not yet implemented")
+            # See #254: Using None as function output leads to [] return value
```

Comment:
You should put the same string in the SkipTest message as there was in the KnownTestFailure message.